remote-facade/src/main/java/com/iluwatar/remotefacade/domain/Domain.java

```diff
@@ -0,0 +1,23 @@
+package com.iluwatar.remotefacade.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class acting as a database to store details in an Arraylist.
+ */
+public class Domain {
```

Comment:
Maybe rename it to Store or DummyDatabase.

Reply:
fixed this